anx/provider/provider.go

```diff
@@ -121,6 +121,11 @@ func (a *anxProvider) initializeLoadBalancerManager(builder cloudprovider.Contro
 }
 
 func (a anxProvider) LoadBalancer() (cloudprovider.LoadBalancer, bool) {
+	if a.loadBalancerManager == nil {
+		a.providerMetrics.MarkFeatureDisabled(featureNameLoadBalancer)
```

Comment:
I think this is not needed, as features start out disabled in `setupProviderMetrics()`